src/Components/Nav/Nav.js

```diff
@@ -1,7 +1,142 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import CommunityTab from './components/CommunityTab';
+import Category from './components/Category';
+import NavAside from './components/NavAside';
+import SigongTab from './components/SigongTab';
+import StoreTab from './components/StoreTab';
 
 function Nav() {
-  return <div></div>;
+  const [categoryShow, setCategoryShow] = useState(false);
+  const [currentId, setCurrentId] = useState(2);
+
+  // useEffect(() => {
+  //   console.log('컨디업');
+  //   if (categoryShow) {
+  //     console.log('true');
+  //   } else {
+  //     console.log('false');
+  //   }
+  //   // console.log(categoryShow);
+  // }, [categoryShow]);
+
+  const handleCategoryOn = () => {
+    setCategoryShow(true);
+  };
+
+  const handleCategoryOff = () => {
+    setCategoryShow(false);
+  };
+
+  const handleMouseEnter = id => {
+    handleCategoryOff();
+    setCurrentId(id);
+  };
+
+  return (
+    <div>
+      <WrapNav>
+        <HeadNav>
+          <NavBar>
+            <NavMain>
+              <div>
+                <Link to="/">
+                  <MainLogo alt="오늘의 집" src="/images/logo.png" />
+                </Link>
+              </div>
+              {CATEGORY_ARR.map((category, idx) => {
+                return (
+                  <NavMainContent>
+                    <Link
+                      key={idx}
+                      className="mainContent-menu"
+                      to="/"
+                      onMouseEnter={() => handleMouseEnter(idx + 1)}
+                    >
+                      {category}
+                    </Link>
+                  </NavMainContent>
+                );
+              })}
+            </NavMain>
+            <NavAside />
+          </NavBar>
+        </HeadNav>
+        {currentId === 1 && <CommunityTab />}
+        {currentId === 2 && (
+          <StoreTab
+            handleCategoryOn={handleCategoryOn}
+            handleCategoryOff={handleCategoryOff}
+          />
+        )}
+        {currentId === 3 && <SigongTab />}
+        {categoryShow && (
+          <Category
+            categoryShow={categoryShow}
+            handleCategoryOff={handleCategoryOff}
+          />
+        )}
```

Comment:
```suggestion
        const tabMapper = {
          1: <CommunityTab />,
          2: <StoreTab setCategoryShow={setCategoryShow} />,
          3: <SigongTab />
        }

...
        {tabMapper[currentId]}
        {categoryShow && (
          <Category
            categoryShow={categoryShow}
            handleCategoryOff={handleCategoryOff}
          />
        )}
```
- 객체에 맵핑해서 쓰면 조금 더 간결해보이겠네요!
- set 함수를 내려주고 아래 컴포넌트에서 on off 함수 만들어 쓰도록 하는건 어떨까요?